explore-analyze/machine-learning/nlp/ml-nlp-model-ref.md

```diff
@@ -15,7 +15,7 @@ products:
 The minimum dedicated ML node size for deploying and using the {{nlp}} models is 16 GB in {{ech}} if [deployment autoscaling](../../../deploy-manage/autoscaling.md) is turned off. Turning on autoscaling is recommended because it allows your deployment to dynamically adjust resources based on demand. Better performance can be achieved by using more allocations or more threads per allocation, which requires bigger ML nodes. Autoscaling provides bigger nodes when required. If autoscaling is turned off, you must provide suitably sized nodes yourself.
 ::::
 
-The {{stack-ml-features}} support transformer models that conform to the standard BERT model interface and use the WordPiece tokenization algorithm.
+The {{stack-ml-features}} support transformer models with the following architectures:
 
 The current list of supported architectures is:
```

Comment:
we don't need both of these because they say the same thing

```suggestion
The {{stack-ml-features}} support transformer models with the following architectures:
```